superduperdb/components/table.py

```diff
@@ -31,6 +31,10 @@ def __post_init__(self, db, artifacts):
         if '_fold' not in self.schema.fields:
             fields.update({'_fold': dtype('str')})
 
+        if '_schema' not in self.schema.fields:
```

Comment:
I don't think this is necessary for `ibis`.

Reply:
yes remove thanks